docs/02.API-REFERENCE.md

```diff
@@ -2021,6 +2036,61 @@ main (void)
 - [jerry_create_typedarray](#jerry_create_typedarray)
 
 
+## jerry_get_container_type
+
+**Summary**
+
+Checks whether the given `jerry_value_t` is the given `jerry_container_type_t` type container object.
+
+*Notes*
+- This API function depends on a build option (`JERRY_ES2015_BUILTIN_CONTAINER`) and can be checked
+  runtime with the `JERRY_FEATURE_MAP, JERRY_FEATURE_SET, JERRY_FEATURE_WEAKMAP, JERRY_FEATURE_WEAKSET`
+  feature enum values.
+  see: [jerry_is_feature_enabled](#jerry_is_feature_enabled).
+- The ES2015-subset profile enables this by default.
+
+**Prototype**
+
+```c
+jerry_container_type_t
+jerry_get_container_type (const jerry_value_t value)
+```
+
+- `value` - Container object
+- return value
+  - The corresponding enum value of `jerry_container_type_t`
```

Comment:
IMHO the doc should mention what happens will be returned if the input value is not a valid container type.